projects/storefrontlib/src/cms-components/checkout/components/payment-method/payment-form/payment-form.component.html

```diff
@@ -191,11 +191,7 @@
       </button>
     </div>
     <div class="col-md-12 col-lg-6">
-      <button
-        class="btn btn-block btn-primary"
-        [disabled]="isContinueButtonDisabled()"
-        (click)="next()"
-      >
+      <button class="btn btn-block btn-primary" (click)="next()">
```

Comment:
did we intentionally drop the disabled state?

Reply:
Yes. The button should always be enabled and the submit button should touch the form to show invalid feedback from the front end side and only submit for backend feedback when the form is valid.